tests/lib/store.sh

```diff
@@ -15,57 +19,63 @@ EOF
     systemctl start snapd.socket
 }
 
+setup_staging_store(){
+    _configure_store_backends "SNAPPY_USE_STAGING_STORE=1"
+}
+
+teardown_staging_store(){
+    if systemctl status snapd.socket; then
+        systemctl stop snapd.socket
+    fi
+    rm -rf $STORE_CONFIG
+    systemctl daemon-reload
+    systemctl start snapd.socket
+}
+
+init_fakestore(){
```

Comment:
not sure it's clear the difference between setup_fake_store and init_fakestore, can we call this one 

init_fake_refreshes ?